code/modules/clothing/under/miscellaneous.dm

```diff
@@ -469,4 +469,21 @@
 	desc = "A preppy purple skirt with a white blouse."
 	icon_state = "plaid_purple"
 	item_state = "plaid_purple"
-	item_color = "plaid_purple"
\ No newline at end of file
+	item_color = "plaid_purple"
+
+/obj/item/clothing/under/whitedress
+	name = "white dress"
+	desc = "A fancy white dress."
+	icon_state = "white_dress"
+	item_state = "w_suit"
+	item_color = "white_dress"
+	fitted = 0

```

Comment:
these use defines now: You want to change the 0 here to NO_FEMALE_UNIFORM. You might want to see if it looks ok with FEMALE_UNIFORM_TOP first though.